tests/mysql/generator_test.go

```diff
@@ -309,6 +312,36 @@ func newActorTableImpl(schemaName, tableName, alias string) actorTable {
 }
 `
 
+var tableSetSchemaFile = `
+//
+// Code generated by go-jet DO NOT EDIT.
+//
+// WARNING: Changes to this file may cause incorrect behavior
+// and will be lost if the code is regenerated
+//
+
+package table
+
+func SetDvdsSchema(schema string) {
```

Comment:
Not sure about name. If schema name appears in the function name, it feels like we are setting something in the schema. I was thinking of `SetSchema` or `UseSchema`.

Reply:
`UseSchema` makes much more sense. Thanks for the input.